lib/grpc/client/adapters/mint/connection_process/connection_process.ex

```diff
@@ -0,0 +1,390 @@
+defmodule GRPC.Client.Adapters.Mint.ConnectionProcess do
+  @moduledoc """
+  This module is responsible for manage a connection with a grpc server.
+  It's also responsible for manage requests, which also includes check for the
+  connection/request window size, split a given payload into appropriate sized chunks
```

Comment:
```suggestion
  connection/request window size, splitting a given payload into appropriate sized chunks
```

Reply:
Fixed in https://github.com/beligante/grpc/pull/1